test/bats/azure.bats

```diff
@@ -310,66 +258,14 @@ setup() {
   assert_success
 }
 
-@test "Test auto rotation of mount contents and K8s secrets - Create deployment" {
```

Comment:
Rotation is tested as part of e2e-provider suite and doesn't need to be done again in azure provider